resources/application-connector/templates/applications-microfrontend.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.managementPlaneUrl -}}
```

Comment:
```suggestion
{{- if .Values.global.managementPlane.url  -}}
```

Reply:
All changes to this file have been reverted. Resolving.